### PR TITLE
Update App Center SDK privacy manifest 

### PR DESCRIPTION
**Pull request by AnatolyPristensky**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Update privacy manifest #2526

## Related PRs or issues

[AB#104086](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/104086)

### CODE REVIEW

**Review by DmitriyKirakosyan — APPROVED** [review]

**Comment by jdheerajcoding** [comment]
@DmitriyKirakosyan Do you happen to know, when this PR could be merged? Kind of waiting on this. Is there a timeline for this?

File: AppCenter.podspec [hunk 1]
```diff
@@ -44,6 +44,7 @@ Pod::Spec.new do |s|
     ss.tvos.frameworks = 'UIKit'
     ss.vendored_frameworks = "AppCenter-SDK-Apple/AppCenter.xcframework"
     ss.libraries = 'z', 'sqlite3'
+    ss.resource_bundle = { 'AppCenter' => 'PrivacyInfo.xcprivacy' }
```
Comment (DmitriyKirakosyan) [inline AppCenter.podspec:47]: I think we don't need this as we distribute binaries which already include privacy manifest.

File: Package@swift-5.3.swift
[the remark this answers is not in the document]
Comment (advantis) [reply Package@swift-5.3.swift:1]: You're right. Perhaps I misunderstood the meaning of "the most compatible tools version". Quick experiment on an empty Swift package shows it indeed picks up a manifest with the highest version among the two.

Anyway, adding the privacy manifest to the `Package.swift` for consistency or removing the extra package manifest (as you suggested) would be a potential improvement.

**Comment by vallgrenerik** [comment]
@AnatolyPristensky Any updates of merging/new release? 😊

**Comment by ushasanaga** [comment]
@AnatolyPristensky can you share the timeline of merging this? We are trying to upgrade to be compliant with Apple's policy.

**Comment by jdheerajcoding** [comment]
@AnatolyPristensky @MikhailSuendukov @DmitriyKirakosyan Any updates on when this PR would be merged/available? Thank you

**Pull request merged**